strategies.py

```diff
@@ -208,37 +196,32 @@ def show_path_to_root(self, node):
 
     def should_resign(self):
         '''Returns true if the player resigned.  No further moves should be played'''
-        if self.root.Q_perspective < self.resign_threshold:  # Force resign
-            self.result = self.root.position.to_play * -2  # use 2 & -2 as "+resign"
-            if self.verbosity > 1:
-                res = "B+" if self.result is 2 else "W+"
-                print("%sResign: %.3f" % (res, self.root.Q), file=sys.stderr)
-                print(self.root.position,
-                      self.root.position.score(), file=sys.stderr)
-            return True
-        return False
+        return self.root.Q_perspective < self.resign_threshold
 
-    def make_result_string(self, pos):
-        if abs(self.result) == 2:
-            res = "B+Resign" if self.result == 2 else "W+Resign"
+    def set_result(self, winner, was_resign):
+        self.result = winner
+        if was_resign:
+            string = "B+R" if winner == go.BLACK else "W+R"
```

Comment:
sort of raises a larger question that we don't have to resolve on this PR, but: will we ever run with integer komi?

Reply:
I think the big bridge to cross there is not even the question of integer komi, but how to make a neural net handle multiple komi types. Agreed that it's sort of weird that ties aren't possible here.

Reply:
did you see  https://arxiv.org/abs/1705.10701 ?

Reply:
Yep, i've read that paper.

Reply:
what did you think of it?